components/admin/invites/autonomous-simulation-page.tsx

```
@@ -0,0 +1,166 @@
+						value={email}
+					/>
+					<textarea
+						className="min-h-40 w-full rounded border p-2"
+						onChange={(event) => setSourceTranscript(event.target.value)}
+						placeholder="Paste transcript"
+						value={sourceTranscript}
+					/>
+					<Button
+						disabled={name.trim().length === 0 || sourceTranscript.trim().length === 0}
+						onClick={() => {
+							void submit();
```

Biome (2.4.4)

[error] The use of void is not allowed.

(lint/complexity/noVoid)